lnbits/core/models.py

```diff
@@ -276,7 +276,7 @@ def from_row(cls, row: Row):
 
 
 class CoreAppExtra:
-    register_new_ext_routes: Callable
+    register_new_ext_routes: Callable = logger.debug
```

Comment:
i dont get this?

Reply:
@motorina0 is this intentional?

Reply:
- `register_new_ext_routes` must not be `None`
- under normal conditions (when the LNbits server is started) this value is initialized. However when starting from `commands` the value was `None`.
- `logger.debug` was used as a default value because it allowed any arguments, but that was a bad idea
- now an explicit empty `_do_nothing(*_)` function is declared and used

Reply:
fixed in commit: https://github.com/lnbits/lnbits/pull/1963/commits/7adcf1545d482ac76f876bb012830c8c5594a998